// @license
// Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
// This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
// The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
// The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
// Code distributed by Google as part of the polymer project is also
// subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt

syntax = "proto2";

package arcs.enclave;

import "asylo/enclave.proto";

message KeyRequest {
  // currently ignored
  optional string value = 1;
}

message KeyResponse {
  optional string key = 1;
}

extend .asylo.EnclaveInput {
  optional KeyRequest arcs_enclave_input = 1000;
}

extend .asylo.EnclaveOutput {
  optional KeyResponse arcs_enclave_output = 1000;
}
